test/sharing/delete_share_test.js

```diff
@@ -45,7 +45,7 @@ describe("Patients", function () {
         // check it requires a valid and authenticated/authorized patient and user
         patients.itRequiresAuthentication(curry(remove)(1));
         patients.itRequiresValidPatientId(curry(remove)(1));
-        patients.itRequiresWriteAuthorization(curry(removePatientShare)({}));
+        patients.itRequiresOnlyMeAuthorization(curry(removePatientShare)({}));
```

Comment:
@dbrezack it's hidden under a few layers of abstraction, but this line does test for the 403 response for attempting to delete a share in every case except when the user owns the share

Reply:
This is weird I reread the code after I originally disapproved this pr and approved it. I don't know why it didn't go through.